src/test/ui/parser/bad-match.stderr

```diff
@@ -1,7 +1,7 @@
 error: expected one of `:`, `;`, `=`, or `@`, found `x`
-  --> $DIR/bad-match.rs:6:13
+  --> $DIR/bad-match.rs:2:13
    |
-LL |   let isize x = 5;
+LL |   let isize x = 5; //~ ERROR expected one of `:`, `;`, `=`, or `@`, found `x`
```

Comment:
For future reference, I prefer to have the error comments just outside of view of the `stderr` files to make it easier to evaluate them in isolation :)

Reply:
Hm, indeed.
That probably means one line below most of the time, like
```rust
let isize x = 5;
//~^ ERROR expected one of `:`, `;`, `=`, or `@`, found `x`
```
?

Reply:
Yeah, that's usually what I do. I only deviate from that when the line below would somehow still be part of the diagnostic, where I add as many `^` as necessary, but there's a point of diminishing returns.

Reply:
>when the line below would somehow still be part of the diagnostic, where I add as many ^ as necessary

We need `//~v ERROR text text text` :smile: